pkg/devfile/adapters/common/generic.go

```diff
@@ -95,7 +99,10 @@ func convertGroupKindToString(exec *common.Exec) string {
 // Init only runs once when the component is created.
 func (a GenericAdapter) ExecDevfile(commandsMap PushCommandsMap, componentExists bool, params PushParameters) (err error) {
```

Comment:
this `commandsMap` is different and we've to keep track, can you help rename it to something else, may prefix with push?